analyses/interaction-plots/scripts/01-disease-specimen-lists.R

```diff
@@ -85,16 +85,15 @@ out_file <- file.path(root_dir, opts$outfile)
 
 #### Read files
 
-meta_df <- readr::read_tsv(meta_file, col_types = readr::cols())
+meta_df <- readr::read_tsv(meta_file, col_types = readr::cols(), guess_max = 10000)
```

Comment:
What is the reason behind adding the `guess_max` argument to the function here?

Reply:
It prevents a warning when readr guesses the wrong data type for some columns (columns that are blank for the first 1000 samples, in this case)